packages/components-js/tests/e2e/puppeteer/specs/select.e2e.ts

```diff
@@ -0,0 +1,1461 @@
+import type { Page } from 'puppeteer';
+import type { Components } from '@porsche-design-system/components';
```

Comment:
Can we import this from src? Otherwise the import can't be found with `components:start`.